README.md

```diff
@@ -74,7 +74,13 @@ When running the patcher on Linux or MacOS, be sure to run the respective binari
 
 For example, on Linux you would run:
 
-`sudo ./linux-x64/Patcher -e=/path/to/Unity -t=dark -linux`
+`sudo ./linux-x64/Patcher -e=/path/to/Unity --t=dark --linux`
 
-NOTE:
+or on Mac:
+
+`sudo ./osx-64/Patcher -e=/Applications/Unity/Hub/Editor/VERSION/Unity.app/Contents/MacOS/Unity --mac --t=dark`
```

Comment:
Maybe format the `VERSION` like `<Version>` or `$VERSION` or something similar so it's a bit more apparent that it is a variable which needs to be replaced before running the Command.

Reply:
Good catch. 👍